heroku.yml

```diff
@@ -1,3 +1,9 @@
-build: 
+setup:
+  addons:
+    - plan: heroku-postgresql
+      as: DATABASE
+  config:
+    HASURA_GRAPHQL_MIGRATIONS_DATABASE_ENV_VAR: DATABASE_URL
```

Comment:
Implementing migrations at deploy time on heroku by pointing to my data ENV that holds the fully qualified URL.